pwndbg/arch.py

```diff
@@ -30,6 +30,7 @@ def fix_arch(arch):
     return next(a for a in arches if a in arch)
 
 @pwndbg.events.start
+@pwndbg.events.new_objfile
```

Comment:
We weren't reloading `pwndbg.arch.endian` when loading new object file (e.g. when connecting to remote target).